umake/network/download_center.py

```diff
@@ -103,7 +106,7 @@ def __init__(self, urls, on_done, download=True, report=lambda x: None):
             future.add_done_callback(self._one_done)
 
     def _fetch(self, download_item, dest):
-        """Get an url content and close the connexion.
+        """Get an url content and close the connection.
```

Comment:
Argh, french/english :p Thanks for the typo fix! (should be in the refactoring/small fixes PR)